python/mxnet/gluon/data/vision/transforms.py

```diff
@@ -257,9 +257,7 @@ class CropResize(HybridBlock):
         interpolation. See OpenCV's resize function for available choices.
         https://docs.opencv.org/2.4/modules/imgproc/doc/geometric_transformations.html?highlight=resize#resize
         Note that the Resize on gpu use contrib.bilinearResize2D operator
-        which only support bilinear interpolation(1). The result would be slightly
-        different on gpu compared to cpu. OpenCV tend to align center while bilinearResize2D
-        use algorithm which aligns corner.
+        which only support bilinear interpolation(1).
```

Comment:
Sorry that I don't know what is `bilinear interpolation(1)`. Does `bilinear interpolation(1)` mean `align_corners=False` ?

Reply:
```align_corners=False/True``` are two different mode of bilinear interpolation or some other interpolation algorithms.
Before this pr:
1. The CPU code of ```nd.Resize``` op use opencv's bilinear interpolation, which is equivalent to BilinearResize2D with argument ```align_corners=False```. 
2. The GPU code of ```nd.Resize``` op use BilinearResize2D internally, but BilinearResize2D implicitly use ```align_corners=True```.
Therefore, I've added the ```align_corners``` argument and add ```align_corners=False``` mode to BilinearResize2D. 
After py*ch 0.4.0, ```align_corners=False``` is the default mode of its interpolate op. Deeplab v3 claims that use ```align_corners=True``` can improve segmentation accuracy. As I've tested, ```align_corners=False``` is more efficient.